selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/OrganizationTest.java

```diff
@@ -60,15 +60,17 @@ public void setUp() throws Exception {
     String firstName = generate("F", 7);
     String lastName = generate("L", 7);
 
-    dashboard.open();
     orgName = generate("orgX", 6);
-
     organization = testOrganizationServiceClient.create(orgName);
+    testOrganizationServiceClient.addAdmin(organization.getId(), adminTestUser.getId());
+
+    dashboard.open(adminTestUser.getName(), adminTestUser.getPassword());
   }
 
   @AfterClass
   public void tearDown() throws Exception {
     testOrganizationServiceClient.deleteById(organization.getId());
+    testOrganizationServiceClient.deleteByName(orgName);
```

Comment:
**deleteByName(orgName)** seems useless so as it is using **deleteById()** inside.